packages/messenger-widget/src/components/Contacts/Contacts.css

```diff
@@ -109,3 +109,50 @@
 .display-name {
     font-weight: bolder;
 }
+
+/* =================== Mobile Responsive CSS =================== */
+
+@media only screen and (max-width: 800px) {
+    .contacts-scroller {
+        padding-left: 0rem !important;
+    }
+
+    .contact-details-container {
+        padding: 0.5rem 0.5rem 0.5rem 2rem !important;
+    }
+
+    .contact-details-container:nth-child(even) {
+        background: var(--alternate-contact-background-color) !important;
+    }
+
+    .contact-details-container:hover {
+        border-radius: 0px;
+        padding-left: 2rem !important;
```

Comment:
Why do we need to use important on every single entry?

Reply:
Its because any other CSS don't override it. We have container queries as well as normal CSS so they don't override it.